src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigWithFineTunedAlignments.java

```diff
@@ -175,11 +184,15 @@ public final boolean hasIncompletePicture() {
     //==================================================================================================================
 
     /**
-     * This predicate tests if an assembly contig has the full event (i.e. alt haplotype) assembled
-     * Of course, the grand problem of SV is always not getting the big-enough picture
-     * but here we have a more workable definition of what is definitely not big-enough:
+     * This predicate tests if an assembly contig with two (picked) alignments has the
+     * <ul>
+     *     <li>full event, or</li>
+     *     <li>breakpoint </li>
+     * </ul>
+     * assembled.
      *
-     * If the assembly contig, with its two (picked) alignments, shows any of the following signature,
+     * If the assembly contig's alignments show any of the following signature,
+     * it will be classified as incomplete.
      * it is definitely not giving the whole picture of the alt haplotype,
```

Comment:
With your edits I'd now start this sentence off, "Incomplete alignments do not give the whole picture.."